src/nrniv/CMakeLists.txt

```diff
@@ -326,9 +326,14 @@ if(NRN_HAVE_NVHPC_COMPILER)
   # For NVHPC we will rely on FE exceptions as opposed to errno in order to make use of faster
   # builtins. One caveat is that if we use an optimization level greater than -O1, the FE exception
   # is not raised. See https://github.com/neuronsimulator/nrn/pull/1930
+  set(NVHPC_MATH_COMPILE_OPTIONS "-O1")
+  if(${CMAKE_C_COMPILER_VERSION} VERSION_GREATER_EQUAL 23.1)
```

Comment:
`CMAKE_C_COMPILER_VERSION` or `CMAKE_CXX_COMPILER_VERSION`?

Reply:
I think this is the same thing for checking the version. We already use `CMAKE_C_COMPILER_VERSION` above